recipes/openimageio/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+
+    def build(self):
+        version = tools.Version(self.deps_cpp_info["openimageio"].version)
+        cmake = CMake(self)
+        cmake.definitions["CMAKE_CXX_STANDARD"] = 14 if version >= "2.3.0.0" else 11
```

Comment:
```suggestion
        cmake = CMake(self)
```

No longer needed now that 2.4.7 is the oldest one available